testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/NebulaPluginsSmokeTest.groovy

```diff
@@ -75,6 +76,12 @@ class NebulaPluginsSmokeTest extends AbstractPluginValidatingSmokeTest implement
 
         then:
         runner('groovydoc', '-s')
+            .expectLegacyDeprecationWarning(
```

Comment:
❓ Is it fixed with the recent version? It looks like we're testing a single version here, I'd expect calling `expectDeprecationWarning` with a follow-up here.

Reply:
No, it's still there. I planned to open a PR to fix it there. Yes, `expectDeprecationWarning` is a proper method to call

Reply:
https://github.com/nebula-plugins/nebula-plugin-plugin/pull/76